From 5875dea90f71350b7205788fbc67e3759077cefe Mon Sep 17 00:00:00 2001
From: Joe MacDonald <joe@deserted.net>
Date: Wed, 20 Nov 2013 16:06:07 -0500
Subject: [PATCH] libpcap: search sysroot for headers

Configure hard-coded host header paths when building with libpcap.  Point
the search path at the sysroot instead.

Upstream-Status: Pending

Signed-off-by: Joe MacDonald <joe@deserted.net>

---
 configure.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.in b/configure.in
index 62573a8..fded45b 100644
--- a/configure.in
+++ b/configure.in
@@ -69,7 +69,7 @@ case "$host" in
     linux="yes"
     AC_DEFINE([LINUX],[1],[Define if Linux])
     AC_SUBST(extra_incl)
-    extra_incl="-I/usr/include/pcap"
+    extra_incl="-I=/usr/include/pcap"
     ;;
   *-hpux10*|*-hpux11*)
     AC_DEFINE([HPUX],[1],[Define if HP-UX 10 or 11])
